
// //faz a mesma coisa que try cacth
// const methodTo = promisse => promisse.then(result => [null, result]).catch(error => [error])

// export default methodTo;